src/jwt.rs

```diff
@@ -711,7 +711,7 @@ pub mod tests {
             .chars()
             .take(TEST_JWT.len() - 1)
             .collect::<String>()
-            + "f";
+            + "w";
```

Comment:
I'm curious why `f` doesn't adequately corrupt the signature

Reply:
I just wrote a small section about that on https://github.com/openid-certification/oidctest/issues/227 so if you want the exact reasoning behind it you may want to read it there as I'm just going for a short version here:

Of course it does adequately corrupt the signature. The problem is that it never arrives at the signature checking as the base64 is invalid thus the decoding fails. Its the same problem that we have on the rs endpoint but there I was able to fix it by replacing it with valid base64. I will reset it once we switch to forgiving parsing